### ISSUE TITLE
materialize.config.scss doesn't work.

### ISSUE THREAD

**Issue by dannyvassallo** [Zevran/materialize-loader#9]
I had my config file nested, so this may not be the solution for everyone -- but I had to remove the semicolons on lines 9 & 10 to get it to load the default color:

```
var styles = [
    'materialize'
];

module.exports = function(content) {
    this.cacheable(true);
    var config = this.exec(content, this.resourcePath);
    var start =
            "$roboto-font-path: \"~materialize-css/fonts/roboto/\";\n"
            + "@import \"~materialize-css/sass/components/_color.scss\";\n"
            + "@import \"~../materialize-config/materialize.config.scss\";\n";
    source = start + styles.filter(function(style) {
        return config.styles[style];
    }).map(function(style) {
        return "@import \"~materialize-css/sass/" + style + ".scss\";";
    }).join("\n");
    return source;
};
```

This fixed it but I obviously have my file in a config folder you see here `"@import \"~../materialize-config/materialize.config.scss\";\n";`

I wouldn't use this as your solution, but I think it may be in everyone's better interest to remove those semicolons since it's breaking the `start` declaration.

**Comment by dannyvassallo** [comment]
Your example is also wrong. The color component names aren't prefixed with "materialize", it should read like this:

```sass
$primary-color: color("red", "lighten-2");
```

And I wouldn't use red because that's the default color, I'd use something that was a more stark or obvious color difference.

**Comment by Zevran** [comment]
Hi @dannyvassallo,

If you want to get your config sass file working you have to put it in the same folder as your config js file specified in your webpack config.

e.g.
```javascript
require("materialize-loader!./src/theme/materialize.config.js");
```
Your ```materialize.config.scss``` have to be in the same folder ```src/theme```

If you don't specify your config js file in webpack config, materialize-loader will use default config files.

Thanks for reporting the error in the example, I will fix this this weekend.

Please tell me if you succeed to get it working without editing materialize-loader files.

**Comment by dannyvassallo** [comment]
@Zevran Thanks for responding!

I dont have my config files in a `src/theme` folder, I created a folder on the root for my materialize config files -- I can move it but the location of the files doesn't seem to really be the issue I'm having. I do have the `materialize.config.js` and `materialize.config.scss` files in the same location as one another, and the relative paths are appropriate throughout my app.

My issue was this. The `materialize.config.scss` file didn't work at all while the `materialize.config.js` file did work when the boolean value of `materialize:` was changed back and forth. Even when I changed the path manually by forking the package and installing from my github, the `.scss` config file didn't work.

When I remove the semicolon that used to be in your start declaration like this:

```
    var start =
    var start =
            "$roboto-font-path: \"~materialize-css/fonts/roboto/\";\n"
            + "@import \"~materialize-css/sass/components/_color.scss\";\n"
            + "@import \"./materialize.config.scss\";\n";
```

versus the way you have it now:

```
    var start =
            "$roboto-font-path: \"~materialize-css/fonts/roboto/\";\n"
            + "@import \"~materialize-css/sass/components/_color.scss\";\n";
            + "@import \"./materialize.config.scss\";\n";
```

The `.scss` file works.

----------

I did switch back to your package and moved my file structure so the configs were nested in `src/theme` and it still doesn't work. The primary color stays the default even when changed. On the other hand, my fork is still working just fine.

I think part of the issue may be the assumption that all of your package's users will know to create a `src` folder, nest a `theme` folder in it and then setup their config -- when there is zero mention of that in the readme and that's not necessarily how everyone is going to setup a node app.

I mean -- I'm not trying to be a jerk, and I could very well be overlooking something in the docs but it seems to me like the package is only half working right now.

**Comment by Zevran** [comment]
I am really sorry, I didn't saw the trailing comma at the end the _color.scss file !
Indeed in that case it can't work as it would.

Well I'll fix it this evening after work and publish a new version.

Again I am terribly sorry for my blindness...

Thanks

**Comment by dannyvassallo** [comment]
@Zevran Don't apologize! You've done some great work here and I'm still using your code! Thanks for your kindness and responsiveness : )

**Comment by Zevran** [comment]
Fixed with #10
